tests/testthat/checkstyle.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<checkstyle version="lintr-1.0.0.9001">
+<checkstyle version="lintr-1.0.1">
```

Comment:
Try to keep unrelated changes out of PRs. If you do include them they should be in separate commits.

Reply:
Sorry about that. And given that you just bumped the version, I think I also created a merge conflict. Should I go ahead and resolve it?